### ISSUE TITLE
Swedish characters not shown in sms

### ISSUE THREAD

**Issue by nikhau** [smokku/pebble#35]
I can read the sms text in Pebble. However, the Swedish characters å, ä and ö are shown as a and o. Fully understandable, but annoying, especially since some words change meaning when replace eg å with a.

Pebble firmware version: 2.8.1
Sailfish Pebble version: 0.15
Sailfish version: 1.1.1.27 (Vaarainjärvi)

**Comment by smokku** [comment]
Pebble firmware supports plain-old ASCII characters only. There's nothing we can do with it.
(There are alternative firmwares with extended character set, but these we can support in one way only - option to disable transliteration we have implemented.)

**Comment by nikhau** [comment]
The Pebble fw 2.8 release introduced multi-language notifications - with full character support for 80 languages, one of which being Swedish. 

For my friend, having an iPhone 6 and official Pebble FW this characters are properly displayed.

**Comment by smokku** [comment]
Oh.
So what happens if you disable transliteration in options?

**Comment by nikhau** [comment]
Disabled at first.
Later Enabled, same behaviour.

**Comment by smokku** [comment]
So it looks like the new firmware transliterates on itself.
If you have any idea how to disable this behavior, let me know.

**Comment by nikhau** [comment]
This is over my head. But looking on watchconnector.h  I see sendSMSNotification(QString sender, QString data). Since the QString class provides a Unicode character string, my only idea is that Pebble maybe expects another codec, utf8 or whatelse, and then something goes wrong when transliterating.

**Comment by nikhau** [comment]
Ok, this is weird. After yesterday's exercising, the Swedish characters suddenly start to work. The only thing changed is that I toggled the transliteration box in the Pebble app.

So this was the way:
Installed Pebble, the box for "transliterate messages" was dark, I assume that equals disabled.
After toggling enabling/disabling, it now works.

Tried then this:
With "translittering messages" dark, the characters are properly displayed.
With "trnaslittering messages" lit, the characters are wrong.

My guess is that, after the fresh install, eventhough the function was disabled (according to the menu setting ui), it was called and transliterated the characters. But after toggling the setting, the app correctly recognized the setting.

Just to clarify; I did not do any reboot, niether phone nor watch, the difference toggling the setting.
